aempractice/bundle/src/main/java/com/adobe/aempractice/impl/MyserviceImpl.java

```diff
@@ -0,0 +1,34 @@
+package com.adobe.aempractice.impl;
+
+import com.adobe.aempractice.MyService;
+
+import org.apache.commons.lang3.*;
+import org.apache.felix.scr.annotations.Component;
+import org.apache.felix.scr.annotations.Service;
+import java.util.logging.Logger;
+@Component
+@Service
+public class MyserviceImpl implements MyService{
+    static Logger logger = Logger.getLogger(MyserviceImpl.class.getName());
+    String abs="";
```

Comment:
Since the method abc  has returning type String so I'm providing the variable. Alternatively null could be used for return too.